js/layout-javascript/news-feed-code.js

```diff
@@ -2437,6 +2453,13 @@ DynamicList.prototype.showDetails = function(id, listData) {
   var wrapper = '<div class="news-feed-detail-wrapper" data-entry-id="{{id}}"></div>';
   var src = _this.src;
 
+  if (!this.$detailsContent || !this.$closeButton) {
+    this.$detailsContent = $('.news-feed-detail-overlay');
+    this.$closeButton = $(_.find(this.$detailsContent.find('.news-feed-detail-overlay-close'), function(element) {
```

Comment:
See https://api.jquery.com/filter/

```suggestion
    this.$closeButton = this.$detailsContent.find('.news-feed-detail-overlay-close').filter(function(i, el) {
      return !$(el).hasClass('tablet');
    });
```